src/types.ts

```diff
@@ -0,0 +1,12 @@
+import StyleDictionary, { Config } from 'style-dictionary'
+
+export type BrandsConfig = {
+  [key: string]: {
+    platforms: string[]
+    themes: string[]
+  }
+}
+
+export type FormatterConfig = {
+  [key: string]: (sd: typeof StyleDictionary, brand: string, theme: string) => Config
```

Comment:
est-ce que ça posait des problèmes de mettre des enums ?

```suggestion
enum Brand {
	PRO: 'pro'
	JEUNE: 'jeune'
}

enum Platform {
  WEB_CSS: 'web.css'
  WEB_TS: 'web.ts'
  MOBILE_TS: 'mobile.ts'
}

enum Theme {
  LIGHT: 'light'
  DARK: 'dark'
}

export type BrandsConfig = {
  [key: Brand]: {
    platforms: Platform[]
    themes: Theme[]
  }
}

export type FormatterConfig = {
  [key: Platform]: (sd: typeof StyleDictionary, brand: Brand, theme: Theme) => Config
```

Reply:
Ca pose pas de pb! C'est ce que j'avais envisagé initialement, mais ça complique l'ajout d'un nouveau thème ou d'une nouvelle brand, puisqu'il faudrait l'ajouter dans l'objet de la config, et dans les types. Est-ce que tu penses que c'est quand meme mieux d'utiliser des enum ?